README.md

```diff
@@ -128,6 +128,48 @@ charge.lastResponse.requestId // see: https://stripe.com/docs/api/node#request_i
 charge.lastResponse.statusCode
 ```
 
+### `request` and `response` events
+
+The Stripe object emits `request` and `response` events.  You can use them like this:
+
+```js
+var stripe = require('stripe')('sk_test_...');
+
+function onRequest(request) {
+  // Do something.
+}
+
+stripe.on('request', onRequest);
+
+stripe.removeListener('request', onRequest);
+// OR: stripe.off('request', onRequest);
+```
+
+#### `request` object
+```js
+{
+  api_version: 'latest',
+  account: 'acct_TEST',       // Only present if provided
+  idempotency_key: 'abc123',  // Only present if provided
+  method: 'POST',
+  path: '/v1/charges'
+}
+```
+
+#### `response` object
+```js
+{
+  api_version: 'latest',
+  account: 'acct_TEST',       // Only present if provided
+  idempotency_key: 'abc123',  // Only present if provided
+  method: 'POST',
+  path: '/v1/charges',
+  status: 402,
+  request_id: 'req_Ghc9r26ts73DRf',
+  elapsed: 445
```

Comment:
Can you add a comment here specifying "milliseconds"? (If that's indeed what it is.)